main.go

```diff
@@ -159,16 +160,21 @@ func main() {
 		dnsClient = dnscontext.NewClient(dnscontext.WithChainContext(ctx), dnscontext.WithCorefilePath(c.CoreDNSConfigPath))
 	}
 
+	var healClient = null.NewClient()
```

Comment:
```suggestion
	var healOptions = []heal.Option{ heal.WithLivenessCheckInterval(c.LivenessCheckInterval),}
```

Reply:
I see your point, but what about c.LivenessCheckTimeout? I suppose it should be added here as well:
var healOptions = []heal.Option{ heal.WithLivenessCheckInterval(c.LivenessCheckInterval),
heal.WithLivenessCheckTimeout(c.LivenessCheckTimeout)}
Or?